doc/user/telstate.rst

```diff
@@ -74,6 +74,10 @@ RFI mask
 Band mask
     :samp:`{acv}_model_band_mask_{type}`
 
+Primary beam
+    :samp:`{acv}_{antenna}_model_primary_beam_{group}_{type}`, where
```

Comment:
Would `acv` contain the antenna name as well?

Reply:
No, ACV is a the name of the antenna-channelised-voltage CBF stream e.g. `wide_antenna_channelised_voltage`.